core/src/test/scala/com/microsoft/azure/synapse/ml/causal/VerifyLinearDMLEstimator.scala

```diff
@@ -0,0 +1,66 @@
+package com.microsoft.azure.synapse.ml.causal
+
+import com.microsoft.azure.synapse.ml.core.test.base.TestBase
+import org.apache.spark.ml.linalg.DenseVector
+import org.apache.spark.ml.regression.{LinearRegression, RandomForestRegressor}
+import org.apache.spark.ml.classification.LogisticRegression
+import org.apache.spark.sql.DataFrame
+
+class VerifyLinearDMLEstimator extends TestBase {
```

Comment:
Extend `EstimatorFuzzing[LinearDMLEstimator]`

Reply:
@memoryz , I got this after the change, 

Type LinearDMLEstimator does not conform to upper bound Estimator[_] with MLWritable of type parameter S

Reply:
figured it out, LinearDMLEstimator has to extend with ComplexParamsWritable

Reply:
Unfortunately, this change doesn't work as it will introduce two new tests, serialization and fuzzing, and they failed.

1. They required treatmentCol and outcomeCol has the default values, it doesn't make sense for DML has the default values.
2. Even I set them default values as "Label", test failed as we throw exception from transform 

resolved as won't fix

Reply:
figured out that when creating TestObject should assign the real parameters to "new LinearDMLEstimator()"